### DISCUSSION TITLE
Reuse README as index page

### DISCUSSION THREAD

**Discussion by ZhiyuanChen** [mkdocstrings/mkdocstrings#523] [Q&A] [answered]
Hi, 

Thank you so much for this amazing work! 

Can I ask if it is possible to use the README file (project & directory) as the index page? 

So that I don't have to have two identical files like [README](https://github.com/ZhiyuanChen/CHANfiG/blob/master/README.md) and [index](https://github.com/ZhiyuanChen/CHANfiG/blob/master/docs/index.md)

**Comment by ZhiyuanChen** [comment]
Apparently you can set the content of `index.md` as
```
--8<-- "path/to/README.md"
```
to make them linked. 

Is there any way so that I don't have to have one additinonal file?

**Comment by pawamoy** [comment, marked as answer]
Hello! This is more a question for MkDocs itself, more than mkdocstrings. The snippets extension is the way to go IMO. You can try to play with symlinks if you really don't want another file, but they have their own issues.

**Reply by ZhiyuanChen** [reply]
Thank you~
